elytron/WFCORE-3947-ocsp-ssl-revocation.adoc

```diff
@@ -0,0 +1,81 @@
+= OCSP certificate revocation
+:author:            Martin Mazanek
+:email:             mmazanek@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-3947[WFCORE-3947]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/ELY-1617[ELY-1617]
+
+=== Dev Contacts
+
+* mailto:mmazanek@redhat.com[Martin Mazanek]
+
+=== QE Contacts
+
+* TBD
+
+=== Affected Projects or Components
+
+* Security
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* New class RevocatingTrustManager
```

Comment:
Just minor, perhaps ```RevocationTrustManager``` is a better name here?

Reply:
Probably yes, will change